My changes (#69)
SW814F20/Report: report/Sprint_2/testPiler.tex

The reason we choose to build it in Dart language is that it is the same language that Flutter uses, but also that the language allows compilation to many targets.
It can be compiled as a stand-alone program for any x86 machine, or it can be transpiled into Javascript, or as is the case with Flutter, it can be compiled to run on Mobile devices. 
So choosing Dart gives us a lot of flexibility in the future if needed.
The disadvantages with using dart is that it is a fairly new language that is not widely adopted.

> This can mean that new commers can have a hardtime understanding how it works and get qualitive learning resources.

Reviewer: ```suggestion
This can mean that newcomers can have a hard time understanding how it works and get qualitative learning resources.
```

> It also means that the language it self is not as mature as e.x. Java or C\#.

Reviewer: ```suggestion
It also means that the language it self is not as mature as Java or C\#.
```

> Another disadvantages is that Dart is a language developed by a private company, meaning if the company decides to stop the development of the language or decides to remove the language completely we would be in troubles.

Reviewer: ```suggestion
Another disadvantages is that Dart is a language developed by a private company, meaning if the company decides to stop the development of the language or decides to remove the language completely, we would be in trouble.
```
